release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -24,16 +24,9 @@ Built on {op-system-base-full} and Kubernetes, {product-title} provides a more s
 You must use {op-system} machines for the control plane, and you can use either {op-system} or {op-system-base} for compute machines.
 //Removed the note per https://issues.redhat.com/browse/GRPA-3517
 
-For {product-title} 4.12 on `x86_64` architecture, Red Hat has added a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months. For {product-title} 4.12 running on 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, the EUS lifecycle remains at 18 months.
+For {product-title} {product-version}, Red Hat offers a 6-month Extended Update Support (EUS) phase that extends the total available lifecycle from 18 months to 24 months on all supported architectures, including x86_64, 64-bit ARM (aarch64), IBM Power® (ppc64le), and IBM Z® (s390x) architectures. Beyond this, Red Hat also offers a 12-month additional EUS add-on, denoted as Additional EUS Term 2, that extends the total available lifecycle from 24 months to 36 months. The Additional EUS Term 2 is also available on all architecture variants of OpenShift Container Platform.
 
-Starting with {product-title} 4.14, each EUS phase for even numbered releases on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, has a total available lifecycle of 24 months.
```

Comment:
"Starting with {product-title} 4.14, each EUS phase for even numbered releases on all supported architectures, including `x86_64`, 64-bit ARM (`aarch64`), {ibm-power-name} (`ppc64le`), and {ibm-z-name} (`s390x`) architectures, has a total available lifecycle of 24 months."

The "even numbered releases" was flagged as important in my original update, so does the 24 months apply to odd numbered releases also?

![Screenshot from 2024-06-12 15-00-08](https://github.com/openshift/openshift-docs/assets/57954076/f4fce632-debd-4115-8671-279ab7144f7b)

In [link](https://access.redhat.com/support/policy/updates/openshift).

Reply:
I'll add that part back.

Reply:
If the purpose of the change is to drop non-4.16 information, seems like the part is not needed

Reply:
The "even numbered" part would be more important if we're dropping "4.16", IMO. It enforces the message that odd numbered releases do not receive this level of support. It would remove any misconceptions.

Reply:
Made another change after consulting with Kathryn.